### ISSUE TITLE
version handling

### ISSUE THREAD

**Issue by vujevits** [MaiaVictor/swarm-js#28]
it would be nice if the versions would be visible on github as well, so it's easy to see which fixes are included in each release. Now one has to look for a version bump commit :)
Also could you release a new version with the latest fixes?

**Comment by evertonfraga** [comment]
Agreed, will make use of github releases with proper release notes.
